app/code/Magento/CatalogImportExport/Model/Import/Product/Type/AbstractType.php

```diff
@@ -342,9 +342,7 @@ protected function attachAttributesById($attributeSetName, $attributeIds)
                         'is_static' => $attribute->isStatic(),
                         'apply_to' => $attribute->getApplyTo(),
                         'type' => \Magento\ImportExport\Model\Import::getAttributeType($attribute),
-                        'default_value' => strlen(
-                            $attribute->getDefaultValue()
-                        ) ? $attribute->getDefaultValue() : null,
+                        'default_value' => $attribute->getDefaultValue() ?: null,
```

Comment:
Hello @anzin 
This fix has already exist in [this PR](https://github.com/magento/magento2/pull/34689/files)
Please roll back this file